src/applications/disability-benefits/view-payments/components/view-payments-lists/helpers.js

```diff
@@ -117,7 +117,9 @@ export const reformatPaymentDates = payments => {
   return payments.map(payment => {
     return {
       ...payment,
-      payCheckDt: moment(payment.payCheckDt).format('MMM D, YYYY'),
+      payCheckDt: moment(payment.payCheckDt).isValid()
+        ? moment(payment.payCheckDt).format('MMM D, YYYY')
+        : '---',
```

Comment:
Would it be better for accessibility to make this "Unknown" or "Unavailable"? A screenreader would completely ignore the `---` (I think).

Reply:
That's not a bad thought. Let me run it by my designer + pm.

Reply:
Great question @micahchiang ! I think "Unavailable" would be a good no data value. Unknown feels a bit too ambiguous, like the correct value was just lost in a dropped fetch request or something.

/cc @peggygannon for awareness and any content guidance.

Reply:
Looping in @sporeboy - I believe he has some concerns with misrepresenting what `---` may mean. It doesn't always mean "unavailable". Additionally, the word "unavailable" seems a little long 🤷 not sure how that will play with table cells consistently.

Reply:
After discussing with @sporeboy, @1Copenut, and @joshkimux, we came to the consensus that using a span with an aria label was the way to go.